src/main/java/org/example/servletsHomework/service/TextService.java

```diff
@@ -0,0 +1,43 @@
+package org.example.servletsHomework.service;
+
+
+import org.example.servletsHomework.dao.TextDao;
+import org.example.servletsHomework.exception.NotFoundException;
+import org.example.servletsHomework.model.Text;
+
+import java.util.List;
+import java.util.Optional;
+
+public class TextService {
+
+    private final TextDao textDao;
+
+    public TextService(TextDao textDao) {
+        this.textDao = textDao;
+    }
+
+    public boolean addText(Long userId, Text text) {
+        return textDao.add(userId, text);
+    }
+
+    public Text getTextById(Long userId, Long textId) {
+        return Optional.ofNullable(textDao.getTextById(userId, textId))
+                .orElseThrow(() -> new NotFoundException("Text not found with id " + textId));
+    }
+
+
+    public List<Text> getAllTexts(Long userId) {
+        return Optional.ofNullable(textDao.getAllTexts(userId))
```

Comment:
No, it is ok that there are no texts for user. You need just respond with empty list. It is valid response. You only need 404 when user try to get specific resource. Text by id for example. In this case you can't respond to him with null. It is not valid value in http. And it is not collection, so you can't respond with empty list. So that responding with not found code.